internal/characters/ganyu/aimed.go

```diff
@@ -45,7 +46,11 @@ func (c *char) Aimed(p map[string]int) action.ActionInfo {
 		HitlagOnHeadshotOnly: true,
 		IsDeployable:         true,
 	}
-
+	var c1cb combat.AttackCBFunc
+	if c.Base.Cons >= 1 {
+		c.c1done = false
```

Comment:
This doesn't need to be a char variable, please stick to the way I described it in the very first comment.